packages/cli-platform-android/src/config/findLibraryName.ts

```diff
@@ -24,7 +24,9 @@ export function findLibraryName(root: string, sourceDir: string) {
     return undefined;
   }
 
-  const match = buildGradleContents.match(/libraryName = ["'](.+)["']/);
+  const match =
+    buildGradleContents.match(/libraryName = ["'](.+)["']/) ??
+    buildGradleContents.match(/libraryName\.set\(["'](.+)["']\)/);
```

Comment:
It would be more explicit and easier to understand if we extracted `match` similar to `buildGradleContents` and populate it under if statements corresponding to `buildGradlePath` and `buildGradleKtsPath` respectively in lines 19 and 21, can we put this logic there and get rid of `??` fallback?

Reply:
Thank you for the great suggestion.
I've made the changes as shown below: https://github.com/react-native-community/cli/pull/2543/commits/4f5eba3fe6cf53ff3f6c8ec60c97f942aa08b99a